packages/website/docs/docs/converting-blocks.md

```diff
@@ -28,6 +28,10 @@ class BlockNoteEditor {
 const HTMLFromBlocks = editor.blocksToHTML(blocks);
 ```
 
+`returns:` The blocks, serialized as an HTML string.
+
+The output is simplified in order to better conform to HTML standards. Block structuring elements are removed, children of blocks which aren't list items are un-nested, and list items are wrapped in `ul`/`ol` tags.
```

Comment:
A user doesn't know what "block structuring elements" are. Unless the user has inspected the HTML structure of the editor, which is unlikely. So Imo `Block structuring elements are removed` can be removed